### ISSUE TITLE
Deprecation in cli package

### ISSUE THREAD

**Issue by msanterre** [vroy/git-gb#9]
Seeing this when running the bin.

```
DEPRECATED Action signature.  Must be `cli.ActionFunc`.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature
```

**Comment by vroy** [comment]
I'm not seeing this warning. Do you think that #10 will fix this?

**Comment by msanterre** [comment]
#10 seems to have fixed it!

**Comment by vroy** [comment]
Closed in #10
